src/rocket.py

```diff
@@ -0,0 +1,19 @@
+import pygame
+
+class Rocket(pygame.sprite.Sprite):
+    def __init__(self, x, y, width, height, color, velocity, acceleration):
+        pygame.sprite.Sprite.__init__(self)
+        self.image = pygame.Surface((width, height))
+        self.image.fill(color)
+        self.rect = self.image.get_rect()
+        self.rect.x = x
+        self.rect.y = y
+        self.velocity = velocity
```

Comment:
velocity, acceleration should be inherited from `LinearKinematics`

Reply:
line this: `self.velocity = LinearKinematics(velocity)`?

Reply:
class Rocket(bla.sprite, LinearKinematics):
...

Gives rocket all public and protected members that linearkinematics has. (Inheritance)